src/stubs/composer.blade.php

```diff
@@ -7,9 +7,9 @@
         "email": "{{ $authorMail }}"
     }],
     "require": {
-        "php": "^7.2",
-        "socialiteproviders/manager": "^4.0",
-        "ext-json": "*"
+        "php": "^7.4 || ^8.0",
```

Comment:
Maybe lets just bump straight to 8 for new providers?

Reply:
👍 Yes totally